tests/clientEncryption/clientEncryption-constants.phpt

```diff
@@ -7,19 +7,23 @@ var_dump(MongoDB\Driver\ClientEncryption::AEAD_AES_256_CBC_HMAC_SHA_512_DETERMIN
 var_dump(MongoDB\Driver\ClientEncryption::AEAD_AES_256_CBC_HMAC_SHA_512_RANDOM);
 var_dump(MongoDB\Driver\ClientEncryption::ALGORITHM_INDEXED);
 var_dump(MongoDB\Driver\ClientEncryption::ALGORITHM_UNINDEXED);
+var_dump(MongoDB\Driver\ClientEncryption::ALGORITHM_RANGE);
 var_dump(MongoDB\Driver\ClientEncryption::ALGORITHM_RANGE_PREVIEW);
 var_dump(MongoDB\Driver\ClientEncryption::QUERY_TYPE_EQUALITY);
+var_dump(MongoDB\Driver\ClientEncryption::QUERY_TYPE_RANGE);
 var_dump(MongoDB\Driver\ClientEncryption::QUERY_TYPE_RANGE_PREVIEW);
 
 ?>
 ===DONE===
 <?php exit(0); ?>
---EXPECT--
+--EXPECTF--
 string(43) "AEAD_AES_256_CBC_HMAC_SHA_512-Deterministic"
 string(36) "AEAD_AES_256_CBC_HMAC_SHA_512-Random"
 string(7) "Indexed"
 string(9) "Unindexed"
-string(12) "RangePreview"
+string(5) "Range"
+%Astring(12) "RangePreview"
```

Comment:
PHP 8.3+ emits deprecation notices when referencing these constants. I'm sure that has to do with using `ZEND_ACC_DEPRECATED` to define the constant, but it's not clear to me why earlier versions don't report anything. I found no mention of this in the [8.3 migration guide](https://www.php.net/manual/en/migration83.php).

In any event, I thought it easier to conditionally ignore the notices instead of split this test into two.